client/src/modules/dashboard/Events/pages/EventPage.tsx

```diff
@@ -24,7 +24,7 @@ export const EventPage: NextPage = () => {
             {loading
               ? 'Loading...'
               : !error
-              ? 'Cant find event :('
+              ? "Can't find event :("
```

Comment:
nit: single quotes instead of double

Reply:
I'd go with a double quote here to avoid escaping the single quote in `Can't`.

Reply:
I was getting a lint error with single + the escaped char so I switched to double

Reply:
Ah right thanks!